functions/_fzf_search_history.fish

```diff
@@ -5,22 +5,26 @@ function _fzf_search_history --description "Search command history. Replace the
         builtin history merge
     end
 
+    if not set --query fzf_history_time_format
+        # Reference https://devhints.io/strftime to understand strftime format symbols
+        set -f fzf_history_time_format "%m-%d %H:%M:%S"
+    end
+
     # Delinate commands throughout pipeline using null rather than newlines because commands can be multi-line
     set -f commands_selected (
-        # Reference https://devhints.io/strftime to understand strftime format symbols
-        builtin history --null --show-time="%m-%d %H:%M:%S │ " |
+        builtin history --null --show-time="$fzf_history_time_format │ " |
         _fzf_wrapper --read0 \
             --print0 \
             --multi \
             --tiebreak=index \
             --prompt="Search History> " \
             --query=(commandline) \
-            --preview="echo -- {4..} | fish_indent --ansi" \
+            --preview="echo -- {} | string replace --regex '^[^│]* │ ' '' | fish_indent --ansi" \
             --preview-window="bottom:3:wrap" \
             $fzf_history_opts |
         string split0 |
         # remove timestamps from commands selected
-        string replace --regex '^\d\d-\d\d \d\d:\d\d:\d\d │ ' ''
+        string replace --regex '^[^│]* │ ' ''
```

Comment:
Would you mind just doing the non-greedy version of what you had? I think it's `.*?`. I think it's more idiomatic and slightly easier to read

Reply:
I realized I was wrong and it didn't work properly. When I had ` │ ` inside a command in my history it removed also the beginning of the command.

Reply:
Yeah but that's probably because you didn't make it non-greedy. That's what the ? does.

Reply:
Ah ok sure, changed it.